src/controllers/index.ts

```diff
@@ -1,6 +1,7 @@
 import 'app/controllers/CheckYourAppealController';
 import 'app/controllers/ConfirmationController';
 import 'app/controllers/EntryController'
+import 'app/controllers/EvidenceUploadController';
```

Comment:
Even smarter might be not register controllers when feature is off compared to use of middleware :) To be tried in the future.

Reply:
great point! 👍